wrappers/kn/src/nativeMain/kotlin/zxingcpp/ImageView.kt

```diff
@@ -5,13 +5,14 @@
 
 package zxingcpp
 
+import cnames.structs.ZXing_Image
 import cnames.structs.ZXing_ImageView
 import kotlinx.cinterop.*
 import zxingcpp.cinterop.*
 import kotlin.experimental.ExperimentalNativeApi
 import kotlin.native.ref.createCleaner
 
-@OptIn(ExperimentalForeignApi::class)
+@OptIn(ExperimentalForeignApi::class, ExperimentalNativeApi::class)
```

Comment:
Why would `ImageView` be suddenly "experimental"?

Reply:
fixed in https://github.com/zxing-cpp/zxing-cpp/pull/764/commits/dbc3617c9f82ac53e94a7e2b2a79a32311dbc542